test/openAPI/features/data_exist.feature

```diff
@@ -1,19 +1,37 @@
-Feature: API endpoint that allows users to check if an entry exists in the Digital Registries database.
-  Request endpoint: POST /data/{registryname}/{versionnumber}/exists
+@method=GET @endpoint=/data/{registryname}/{versionnumber}/exists
+Feature: API endpoint that allows users to check if an entry exists in the database.
 
-  Scenario: The user receives a message that the record exists in the Digital Registries database
-    Given The user wants to check if a record exists in the Digital Registries database
-    When The user sends a valid request to check if a record exists in the database
-    Then The process of receiving a message that a record exists in the database completes successfully
-    And The user receives the information that a record exists in the database
+  @smoke
+    Scenario: Successfully receives a message that the record exists in database
+    Given get the database users information of Digital Registries
+    When POST request with given path params "registryname" as registryname and "111" as versionnumber
```

Comment:
specify what is called. we can repeat it in .feature file, but if we do so repeating code for it in .js file will compile with errors.
So better make the text more unique

Reply:
+ for each given when then it works like that